apps/chat/bots.py

```diff
@@ -51,8 +51,12 @@ class TopicBot:
         conversation history of the participant's chat with the router / main bot.
     """
 
-    def __init__(self, session: ExperimentSession, experiment: Experiment | None = None):
+    def __init__(
+        self, session: ExperimentSession, experiment: Experiment | None = None, disable_tools: bool | None = False
+    ):
         self.experiment = experiment or session.experiment
+        if disable_tools:
+            self.experiment.tools = []
```

Comment:
Rather than mutating the experiment itself could we not pass this down to `create_experiment_runnable`?

We'd also want to do it for 'child' nodes right?